### PR TITLE
times-newer-roman: init at 2018-09-11

### PR DESCRIPTION
**Pull request by samuelgrf**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Synthetica9** [comment]
Just an idea seeing this PR, not necessarily related: shouldn't fonts be in a separate attribute under pkgs? Since you can't really install them using `env.systemPackages`

**Comment by samuelgrf** [comment]
@Synthetica9 Sounds like a good idea, this would also make it easier to search for font packages.

File: pkgs/data/fonts/times-newer-roman/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ lib, fetchzip }:
+
+let
+  version = "unstable-2018-09-11";
+in
+fetchzip {
+  name = "times-newer-roman-${version}";
+
+  url = "https://timesnewerroman.com/assets/TimesNewerRoman.zip";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/times-newer-roman/default.nix:9]: Can we upload this to web.archive.org to get a stable URL?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/times-newer-roman/default.nix (hunk 1 above)
Comment (samuelgrf) [reply pkgs/data/fonts/times-newer-roman/default.nix]: @SuperSandro2000 Done!

**Pull request merged**